### PR TITLE
update to quickstart and endpoints

### PR DESCRIPTION
**Pull request by theoriginalstove**

### CODE REVIEW

File: argumenting.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+`url` field should look something like: 
+POST `https://api.sampleapis.com/coffee/iced`
+Body should have: 
+`{
+ "name": {name} 
+ "cost": {cost}
+ } 
+`
+Then in arguments, 
+have a required argument with the `name` being `name` and a second one with name being `cost`
```
Comment (thethan) [inline argumenting.mdx:29]: probably a me doing ... but change the `with the name` to be not have ticks

**Review by thethan — APPROVED** [review]
Overall looks good just one little change suggestion

File: endpoints.mdx [hunk 2]
```diff
@@ -4,20 +4,17 @@ description: 'Learn how to create and manage endpoints to access your Canopies'
 ---
 
 <Info>
-    Endpoints are created in order to access your Canopies through HTTP calls.
+    Canopy Endpoints are created in order to access your Canopies through HTTP calls. 
 </Info>
 
-<Tabs>
```
Comment (alix-ryu-canopy) [inline endpoints.mdx:10]: I like tabs, but I agree that for consistency-sake, it'll be better to do what you did and go with sections.

**Review by alix-ryu-canopy — APPROVED** [review]
Looks good!

File: argumenting.mdx (hunk 1 above)
Comment (theoriginalstove) [reply argumenting.mdx:29]: okay I can remove those. I straight up copied your readme we had laying around.

Comment (thethan) [reply argumenting.mdx:29]: yeah just remove the backticks

**Pull request merged**